modules/environment-roles/templates/shared_iam_terraform_policy.json.tpl

```diff
@@ -53,6 +47,7 @@
         "arn:aws:iam::${account_id}:policy/TDRSNSPublish${title(environment)}",
         "arn:aws:iam::${account_id}:policy/TDRVpcFlowlogPolicy${title(environment)}",
         "arn:aws:iam::${account_id}:policy/TDRYaraAvPolicy",
+        "arn:aws:iam::${account_id}:policy/UpdateWAFAndSecurityGroupsPolicy${title(environment)}",
```

Comment:
Rename with `TDR` prefix?

Reply:
I've done this one and the role as well and moved them so they're alphabetical.